Sender.cpp

```diff
@@ -0,0 +1,69 @@
+#include <string.h>
+#include "Sender.h"
+#include "SensorConstants.h"
```

Comment:
About sensor constants, are these id definitions (0, 1) related to Sensor1 and Sensor2 Test Samples files correct? i think if is better to standardize the Sensor id to 0,1 and same inside the filename of test samples